README.md

```diff
@@ -199,7 +199,41 @@ Besides `PARK_API_CONVERTER`, there are a few other configuration options to set
 
 1) You can create a config file called `config.yaml` in your root folder.
 2) You can set any config value by env var, but you have to prefix the config name with `PARKAPI_` then. For example,
-   you can configure `SECRET_KEY` using the env var `$PARKAPI_SECRET_KEY`. ENV vars overwrite
+   you can configure `SECRET_KEY` using the env var `PARKAPI_SECRET_KEY`. ENV vars overwrite
```

Comment:
[nitpick]

Since you're modifying this line already, what do env vars overwrite?